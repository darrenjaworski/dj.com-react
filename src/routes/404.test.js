import React from "react";
import renderer from "react-test-renderer";

import FourOhFour from "./404";

describe("FourOhFour", () => {
  const img = "https://www.fillmurray.com/g/400/400";
  it("renders without crashing", () => {
    const tree = renderer.create(<FourOhFour image={img} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
