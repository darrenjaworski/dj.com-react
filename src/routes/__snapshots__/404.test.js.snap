// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FourOhFour renders without crashing 1`] = `
Array [
  <h1
    className="css-lozscu"
  >
    Human sacrifice, dogs and cats living together... mass hysteria!
  </h1>,
  <img
    alt="Bill Murray."
    className="css-ks7ago"
    src="https://www.fillmurray.com/g/400/400"
  />,
]
`;
